packages/pakistan/README.md

```diff
@@ -0,0 +1,181 @@
+# 🇵🇰 Paksitan
+
+Contains all the provinces of Pakistan:
```

Comment:
Could you provide only the provinces or only the districts please? Otherwise the `<path>`s are overlapped and the components cannot target a province

Reply:
Ahan, I made a boo boo here, with fill (none) they were just borders and it doesn't make sense now so yep, i am removing it.